crates/apps/src/bin/namada/cli.rs

```diff
@@ -58,7 +57,7 @@ fn handle_command(cmd: cli::cmds::Namada, raw_sub_cmd: String) -> Result<()> {
         }
         cli::cmds::Namada::Wallet(_) => handle_subcommand("namadaw", sub_args),
         cli::cmds::Namada::Relayer(_) | cli::cmds::Namada::EthBridgePool(_) => {
```

Comment:
`cli::cmds::Namada::EthBridgePool` is still reachable from `namadac` I think, so we need to handle it separately

Reply:
within this PR or in a different one? Also, how should this be handled?